src/progressbar/progressbar.ts

```diff
@@ -0,0 +1,49 @@
+import {Component, Input} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-progressbar',
+  template: `
+    <progress class="progress {{striped && 'progress-striped'}} {{type && 'progress-' + type}}" [value]="value" [max]="max">
+      <div class="progress">
+        <span class="progress-bar" [style.width.%]="calculatePercentage()"><ng-content></ng-content></span>
+      </div>
+    </progress>
+  `
+})
+export class NgbProgressbar {
+  private _max = 100;
+  private _striped = false;
+  private _value: number;
+  min = 0;
+
+  @Input()
+  set max(value: number | string) {
+    this._max = parseInt(`${value}`, 10);
+    this.calculatePercentage();
+  }
+
+  get max(): number | string { return this._max; }
+
+  @Input()
+  set striped(value: boolean | string) {
+    if (value === '') {  // boolean usage
+      value = true;
+    }
+
+    this._striped = !!value;
+  }
+
+  get striped(): boolean | string { return this._striped; }
+
+  @Input() type: string;
+
+  @Input()
+  set value(value: number) {
+    this._value = value;
+    this.calculatePercentage();
+  }
+
+  get value(): number { return this._value; }
+
+  private calculatePercentage() { return 100 * this.value / (this._max - this.min); }
```

Comment:
private thing is a leftover (we, I mean I, changed conventions lots of times lately :P)

Not sure about the getter. Will think.